lib/view/icu_non_intensivist/ventilation/ventilation_view.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:wh_covid19/style.dart';
+import 'package:wh_covid19/widget/reusable_card.dart';
+
+import '../../../routes.dart';
+
+class VentilationView extends StatelessWidget {
+  final title = 'Ventilation';
+  final cards = <ReusableCard>[
+    ReusableCard(
+      title: 'Suggested initial actions',
+      color: Colors.white,
```

Comment:
I think we are trying to  only use colors defined in style.dart

Reply:
@maks There isn't any white in style.dart. This is for white cards. I was following the pattern in `staff_welfare_view` and `InfoView` https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/09958dca38fa549ad002f185051b13052c06142b/lib/view/info_view.dart#L10